declare.py

```diff
@@ -150,6 +139,7 @@ def read_credentials():
         type=str,
         help="whether someone in the same household with symptoms - 'Y' or 'N'. defaults to no",
         default="N")
+    parser.add_argument("--no-random-timer", action="store_true")
```

Comment:
Cool feature! Not sure why any legitimate user would have a use for this but I guess it's fine to merge! ;)